src/components/PickerDay.vue

```diff
@@ -331,7 +335,8 @@ export default {
         'sat': day.isSaturday,
         'sun': day.isSunday,
         'highlight-start': day.isHighlightStart,
-        'highlight-end': day.isHighlightEnd
+        'highlight-end': day.isHighlightEnd,
+        ...this.dayCustomClasses(day)
```

Comment:
Why the spread operator here? Isn't is just one function with variable output?

Reply:
Please see https://github.com/charliekassel/vuejs-datepicker/pull/548/files#diff-76ec76da861b1b1cdc6bba450fc735c2R370,
an object with key-value pairs is returned